### ISSUE TITLE
[Tags] Feature request: support formatting in tag descriptions

### ISSUE THREAD

**Issue by dcsjapan** [flarum/core#802]
It might be nice to support Markdown and/or BBcode in tag descriptions, as per [this discussion](https://discuss.flarum.org/d/1992-formatting-tag-description-text).

Considerations:

It may be necessary to limit the types of formatting that can be used in descriptions (assuming that's even possible). However things such as bold, italics, and links would come in handy.

**Comment by Dakoom** [comment]
Yes, I agree on this. A lot of times I've seen things like "Check the rules (link) of this section" on descriptions.

**Comment by fkndean** [comment]
Is this still being considered? I'd like to do some bbcode or use markdown in the tag description. Would be very useful 👍

**Comment by tobscure** [comment]
I think simply allowing HTML might be best, for two reasons:
* More flexibility, which is a good thing when it's the administrator who's the one editing tag descriptions
* BBCode and Markdown are extensions in themselves, and we want to avoid extension-extension interactions where possible

**Comment by tobscure** [comment]
Yes!

Could still be used in `<meta name="description">` tag though, just run it through `striptags` beforehand. Happy to accept a PR!

**Comment by johannsa** [comment]
@tobscure Thanks. I will send a PR as soon as it is ready.

Now, going back to the issue here, should not this issue be closed now that...?
- Hero will be gone and description won't be shown natively (?)
  - Except on `<meta name="description">` once the PR is merged.
  - (?) Can you confirm this?
- Still, you can enter code on the description field.
  - Should this be restricted to text only via validation?

**Comment by tobscure** [comment]
Description will still be shown natively. There'll be some new UI when viewing a tag in place of the hero.

**Comment by franzliedke** [comment]
More semantic HTML sanitization (i.e. allow only a predefined list of tags) may be desirable, too, so that the page cannot be broken completely.

**Comment by ainar** [comment]
Hey! I'm looking for this feature. Any update?

**Comment by askvortsov1** [comment]
It's on the eventual to-do, but isn't a priority at this point as the team is focused on getting to a stable release
